compiler/passes/errorHandling.cpp

```diff
@@ -799,10 +799,18 @@ bool ErrorCheckingVisitor::enterCallExpr(CallExpr* node) {
             inThrowingFunction = parentFn->throwsError();
           }
           if (mode == ERROR_MODE_STRICT) {
-            USR_FATAL_CONT(node, "throwing call without try or try! (strict mode)");
+            USR_FATAL_CONT(node, "call to throwing function %s "
+                                 "without try or try! (strict mode)",
+                                 calledFn->name);
+            USR_PRINT(calledFn, "throwing function %s defined here",
```

Comment:
Some ideas for this second line, though I'm not particularly invested:
- "throwing function" is redundant because it's already called a throwing function in the `FATAL_CONT`
- indent to associate it with the above line

```
implicit-relaxed-try-nothrows-inner.chpl:6: error: call to throwing function throwAnError without throws, try, or try! (relaxed mode)
  ./ThrowError.chpl:1: note: throwAnError defined here
```

Reply:
@lydia-duncan: Feel free to also reject these suggestions if you think they're off-base.

Reply:
I think it'd be new & different to indent the 2nd line (AFAIK no other compiler error/warning does this). I think it's reasonable to repeat 'throwing function' in this case but I'd be OK removing it from the 'defined here' line if you & Lydia both view it as an improvement.

Reply:
To the 2nd line: ah okay, wasn't sure of what was typical.

To repeating throwing function: that's reasonable too, ignore me.

Reply:
I think this error message is fine as is.  I'm okay with repeating that the function is throwing, to ensure the connection between the two lines is obvious, but I don't feel strongly enough to object if it was removed.

Reply:
I agree the indentation looks a little odd, but I think it would be unnecessarily complicated to change